src/extension.ts

```diff
@@ -89,52 +95,31 @@ export async function activate(context: vscode.ExtensionContext) {
         })
         await ext.telemetry.start()
 
-        registerCommand({
-            command: 'aws.login',
-            callback: async () => await ext.awsContextCommands.onCommandLogin(),
-            telemetryName: 'aws_credentialslogin'
-        })
-
-        registerCommand({
-            command: 'aws.credential.profile.create',
-            callback: async () => await ext.awsContextCommands.onCommandCreateCredentialsProfile(),
-            telemetryName: 'aws_credentialscreate'
-        })
+        // TODO remove logout and rename login, they make no sense from a UX standpoint
+        vscode.commands.registerCommand('aws.login', async () => await ext.awsContextCommands.onCommandLogin())
+        vscode.commands.registerCommand('aws.logout', async () => await ext.awsContextCommands.onCommandLogout())
```

Comment:
I'd say "login" should be re-worded to 'connect' - both imply a session but, at least for me personally, when I login to something it's much less ephemeral. A 'connection' usually controls its own 'disconnection' without requiring an explicit command.

FWIW - I agree that 'logout' makes no sense - there is no persistent session.